spec/services/message_employee_matcher_spec.rb

```diff
@@ -5,7 +5,7 @@
   describe "#run" do
     context "onboarding messages (sent days after start)" do
       it "matches messages to users based on days after start" do
-        Timecop.freeze(Time.parse("11:00:00 UTC")) do
+        Timecop.freeze(Time.parse("0:00:00 UTC")) do
```

Comment:
we have a context below that is context "midnight UTC" do` -- should we combine these specs into that context? 

trying to think about how we can make it obvious that our tests cover this case so regressions aren't added in the future.